### PR TITLE
Updating high-level architecture documentation

### PR DESCRIPTION
**Pull request by rdoxenham**

### CODE REVIEW

**Comment by rdoxenham** [comment]
Note that this will need Joe's contributions on SUMA and the LCM/Upgrade Controller stuff to merge first as we're currently failing tests due to missing sections.

File: asciidoc/edge-book/welcome.adoc [hunk 1]
```diff
@@ -9,29 +9,75 @@ ifdef::env-github[]
...
+  ** Multi-cluster management with <<components-rancher,Rancher Prime>>, enabling a common dashboard for downstream cluster onboarding and ongoing lifecycle management of infrastructure and applications, also providing comprehensive tenant isolation and `IDP` (Identity Provider) integrations, a large marketplace of third-party integrations and extensions, and a vendor-neutral API.
+  ** Linux systems management with SUSE Manager, enabling automated Linux patch and configuration management of the underlying Linux operating system (* <<components-slmicro,SLE Micro>>) that runs on the downstream clusters. Note that while this component is containerized, it currently needs to run on a separate system to the rest of the management components, hence labelled as "Linux Management" in the diagram above.
+  ** A dedicated <<components-upgrade-controller,Lifecycle Management>> controller that handles management cluster component upgrades to a given SUSE Edge release.
+  ** Remote system onboarding into Rancher Prime with <<components-elemental,Elemental>>, enabling late binding of connected edge nodes to desired Kubernetes and application deployment, e.g. via GitOps.
```
Comment (e-minguez) [inline asciidoc/edge-book/welcome.adoc:40]:
```suggestion
  ** Remote system on-boarding into Rancher Prime with <<components-elemental,Elemental>>, enabling late binding of connected edge nodes to desired Kubernetes clusters and application deployment, e.g. via GitOps.
```

**Review by e-minguez — COMMENTED** [review]
Small nits.

**Review by e-minguez — DISMISSED** [review]
LGTM

**Review by atanasdinov — DISMISSED** [review]

File: asciidoc/edge-book/welcome.adoc [hunk 2]
```diff
@@ -43,12 +89,17 @@ For customers that need to operate in standalone, air-gapped, or network limited
...
+All SUSE Edge releases comprise of tightly integrated and thorougly validated components that are versioned as one. As part of the continuous integration and stack validation efforts that not only test the integration between components but ensure that the system performs as expected under forced failure scenarios, the SUSE Edge team publishes all of the test runs and the results to the public. The results along with all input parameters can be found at https://ci.edge.suse.com[ci.edge.suse.com].
+
+== Full Component List
 
-SUSE Edge is comprised of both existing SUSE components, including those from the Linux and Rancher teams, along with additional features and components built by the Edge team to enable SUSE to address both the infrastructure requirements and intricacies. The list of components, along with a link to a high-level description of each and how it's used in SUSE Edge can be found below:
+The full list of components, along with a link to a high-level description of each and how it's used in SUSE Edge can be found below:
 
 * <<components-rancher,Rancher>>
 * <<components-rancher-dashboard-extensions,Rancher Dashboard Extensions>>
+* SUSE Manager
```
[the remark this answers is not in the document]
Comment (rdoxenham) [reply asciidoc/edge-book/welcome.adoc:102]: Yes, for sure, we should.

**Review by atanasdinov — APPROVED** [review]

**Pull request merged**